app/packs/review/_components/UnreviewedPack.tsx

```diff
@@ -8,41 +18,199 @@ import {
 	CardHeader,
 	CardTitle
 } from '@/components/ui/card'
-import { UnreviewedPackDetails } from './UnreviewedPackDetails'
+import { QuestionPackDetails } from '../../_components/QuestionPackDetails'
+import { Textarea } from '@/components/ui/textarea'
+import { toast } from '@/components/ui/use-toast'
+import { Check, X, Loader2 } from 'lucide-react'
+import { Button } from '@/components/ui/button'
+import { Badge } from '@/components/ui/badge'
+import { packMap } from '@/types'
+import { useState } from 'react'
+
+import { PackType } from '@/types'
```

Comment:
As part of the below comment on the interface we can change this to be from @prisma/client
```suggestion
import { PackType } from '@prisma/client'
```